modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -1,31 +1,23 @@
 # frozen_string_literal: true
 
 require 'sidekiq'
-require 'va_forms/regex_helper'
 
 module VAForms
   class FormReloader
     include Sidekiq::Worker
-    include SentryLogging
-    FORM_BASE_URL = 'https://www.va.gov'
+
+    # Only retry for ~30 minutes since the job that spawns this one runs every hour
+    sidekiq_options retry: true, unique_for: 30.minutes
+
+    BATCH_SIZE = 100
 
     def perform
       Rails.logger.info("#{self.class.name} is being called.")
-      query = File.read(Rails.root.join('modules', 'va_forms', 'config', 'graphql_query.txt'))
-      body = { query: }
-      response = connection.post do |req|
-        req.path = 'graphql'
-        req.body = body.to_json
-        req.options.timeout = 300
-      end
-      forms_data = JSON.parse(response.body)
-      forms_data.dig('data', 'nodeQuery', 'entities').each do |form|
-        build_and_save_form(form)
-      rescue => e
-        message = "#{self.class.name} failed to import #{form['fieldVaFormNumber']} into forms database"
-        Rails.logger.error(message, e)
-        log_message_to_sentry(message, :error, body: e.message)
-        next
+
+      Sidekiq::Batch.new.jobs do
+        all_forms_data.each_slice(BATCH_SIZE) do |form|
+          Sidekiq::Client.push_bulk('class' => VAForms::FormBuilder, 'args' => form)
```

Comment:
@cilestin You had some concerns about slamming all the jobs at once. I ended up going with batching, but I had also considered a 1 second perform delay between each of the ~700 jobs. You cool with this?

Reply:
I know your comment was meant for @cilestin and not me, but I think we may want to stagger them.

Reply:
I'm not sure if Drew is concerned with the sidekiq queuing or the concurrent execution of 700 sidekiq jobs around the same time.

For the queuing, push_bulk is documented as "useful if you are pushing tens of thousands of jobs or more", and we are only calling it in batches of 100, so it sounds ok, to me.

On the execution side, I _imagine_ out of the 700 urls we fetch, some of them hit the same server.  I think we would need to have a look at the urls and see if we notice any hot urls that could potentially hit with a large number of concurrent PDF fetches.....if we do have some hot urls, perhaps consider grouping the urls and queue them using perform_in to stagger out the individual PDF fetches over time so we don't negatively impact the PDF host server with multiple concurrent requests.

Just my 2cents

Reply:
@michelpmcdonald has it correct - I'm more worried about particular servers getting hit with concurrent requests for PDFs. Sidekiq should easily be able to handle 700 jobs being made at once, it's what those 700 jobs might do at nearly the exact same time that's the "concern." 

While I don't think it should truly be an issue (we're hitting public VA.gov links and if it can't handle 700 requests at the same time it's got some other issues going on), we should at least be cognizant that staggering is an option if we start seeing issues.

To be clear, I do not think this is something we need to worry about for this PR - we can watch it and see if staggering is something we need to do later down the line. Let's start simple now w/o worrying about it and add it later if we need 😄